examples/gallery/images/README.txt

```diff
@@ -0,0 +1,2 @@
+Images, contours, and fields
```

Comment:
![image](https://user-images.githubusercontent.com/23487320/110893549-16a35e80-835b-11eb-9cb3-70f1e3690397.png)

Just on this, I don't quite get the 'contours' and 'fields' part which seem to be from matplotlib. There's no contours nor fields involved here, so could we just do 'Images and grids' or something else?

Reply:
See my comment at https://github.com/GenericMappingTools/pygmt/issues/876#issuecomment-791649980.

> This category mainly focuses on the illustration of matrix data (e.g., images and grids)
> 
> - grdimage: plot a grid by color-coded pixels
> - grdvector: plot a grid by vectors or fields
> - grdcontour: plot a grid by contours
> - image: raster image is just a special matrix data
> - contour: never used it, but I think it's similar to grdcontour

If we follow the categories of my comment, then we will have "fields" examples in this category.

Reply:
Ah ok, so just having it as a placeholder for future examples then. Sounds good.